src/main/java/seedu/address/logic/commands/CalendarCommand.java

```diff
@@ -0,0 +1,63 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.date.AthletickDate;
+
+/**
+ * Allows user to navigate the date of the calendar and view training and performance details on a
+ * specified date.
+ */
+public class CalendarCommand extends Command {
+
+    public static final String COMMAND_WORD = "calendar";
+
+    public static final String MESSAGE_USAGE = "calendar" + ": Jumps to a specified date."
+            + " Specify month and year to view calendar for that month. Specify day, month and "
```

Comment:
Can consider putting the example after the respective outcome since there are two kinds of inputs here.

Reply:
Good suggestion!